responsibleai/tests/common_utils.py

```diff
@@ -108,6 +108,51 @@ def create_boston_data():
     return X_train, X_test, y_train, y_validation, boston.feature_names
 
 
+def create_adult_income_dataset():
+    from dice_ml.utils import helpers
+    dataset = helpers.load_adult_income_dataset()
+    continuous_features = ['age', 'hours_per_week']
+    target_name = 'income'
+    target = dataset[target_name]
+    classes = list(np.unique(target))
+    categorical_features = list(set(dataset.columns) -
+                                set(continuous_features) -
+                                set([target_name]))
+    # Split data into train and test
+    data_train, data_test, y_train, y_test = train_test_split(
+        dataset, target,
+        test_size=0.2, random_state=7, stratify=target)
+    return data_train, data_test, y_train, y_test, categorical_features, \
+        continuous_features, target_name, classes
+
+
+def create_complex_classification_pipeline(
+        X_train, y_train, continuous_features, categorical_features):
+    from sklearn.compose import ColumnTransformer
```

Comment:
Can these imports be at the module level?

Reply:
Done.